modules/Material/Device.qml

```diff
@@ -22,12 +22,12 @@ pragma Singleton
 Object {
     id: device
 
-    // TODO: Actually set this up correctly
-    property string formFactor: "desktop"
-
-    // TODO: Actually set this up correctly
-    property string iconName: "hardware/computer"
```

Comment:
Can you add the iconName and name properties back in, and bind them to the device type?